public/leaderboardinfo.php

```diff
@@ -50,28 +45,16 @@
 $numLeaderboards = getLeaderboardsForGame($gameID, $allGameLBData, $user);
 $numArticleComments = getArticleComments(ArticleType::Leaderboard, $lbID, 0, 20, $commentData);
 
-$errorCode = requestInputSanitized('e');
-
-RenderHtmlStart(true);
-?>
-<head prefix="og: http://ogp.me/ns# retroachievements: http://ogp.me/ns/apps/retroachievements#">
-    <?php RenderSharedHeader(); ?>
-    <?php RenderOpenGraphMetadata(
+RenderOpenGraphMetadata(
     $pageTitle,
     "Leaderboard",
     "$gameIcon",
     "/leaderboardinfo.php?i=$lbID",
     "Leaderboard: $lbTitle ($gameTitle, $consoleName): "
-); ?>
-    <?php RenderTitleTag($pageTitle); ?>
-</head>
-<body>
-<?php RenderHeader($userDetails); ?>
-
+);
+?>
```

Comment:
Doesn't need it but it's much cleaner to have it in there 👍 
`RenderOpenGraphMetadata()` already takes care of setting the page title but `RenderContentStart` should still be present in case we'd want to wrap all the legacy scripts in some custom markup at some point.
I think the scripts will all be migrated to views and controllers before we'll ever need a wrapper. Clean is clean, added where missing.